docs/advanced/UsageWithReactRouter.md

```diff
@@ -1,4 +1,158 @@
 # Usage with React Router
 
-Sorry, but we're still writing this doc.  
-Stay tuned, it will appear in a day or two.
+So you want to do routing with your Redux app. You can use it with the react-router library. Redux will be the source of truth for your data and react-router will be the source of truth for your URL. In most of the cases, **it is fine** to have them separate unless if you need to time travel and rewind actions that triggers the change URL.
+
+## Redux-router, react-router-redux, react-router
+Before starting, let's clarify the different routing libraries.
+
```

Comment:
In fact I’d like to completely deemphasize these other libraries here and maybe add them as a footnote. Let’s focus on RR instead.